Info about admin api v2 vs v3 (#3707)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-cloud-apis/admin-cloud-api.md

- Manage your dbt Cloud account
- and more

> dbt Cloud currently supports two versions of the Administrative API: v2 and v3. In general, v3 is the recommended version to use, but we don't yet have all our v2 routes upgraded to v3. We're currently working on this. If you can't find what you're looking for in our v3 docs, check out the shorter list of v2 endpoints and see if that helps. 

Reviewer: Just suggestions to tighten the wording a tiny bit:
```suggestion
dbt Cloud currently supports two versions of the Administrative API: v2 and v3. In general, v3 is the recommended version to use, but we don't yet have all our v2 routes upgraded to v3. We're currently working on this. If you can't find something in our v3 docs, check out the shorter list of v2 endpoints because you might find it there. 
```
